packages/lodestar-cli/test/unit/config/beaconParams.test.ts

```diff
@@ -17,6 +17,7 @@ describe("config / beaconParams", () => {
   const testCases: {
     id: string;
     kwargs: Parameters<typeof getBeaconParams>[0];
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
it feels to me like we are disabling this particular rule in a lot of places in this PR.  should we maybe just disable it at the eslint config level?

Reply:
I had briliant solution to allow snake case if property is quoted but prettier doesnt let me